crypto/provider.c

```diff
@@ -21,7 +21,7 @@ OSSL_PROVIDER *OSSL_PROVIDER_load(OPENSSL_CTX *libctx, const char *name)
         && (prov = ossl_provider_new(libctx, name, NULL)) == NULL)
         return NULL;
 
-    if (!ossl_provider_activate(prov)) {
+    if (!ossl_provider_activate(prov, libctx)) {
```

Comment:
Oh!  This didn't show when I looked at the commit that implements this change...

Reply:
... but hold on, I want to look a bit further...  I'm still not convinced this is needed specifically for the FIPS module.